specs/subresourceintegrity/spec.markdown

```diff
@@ -595,15 +577,15 @@ failed resource with a different one.
 
 Whenever a user agent attempts to [obtain a resource][] pointed to by a
 `link` element that has a `rel` attribute with the value of `stylesheet`,
-insert the following step between steps 1 and 2:
+modify step 4 to read:
 
-1.  Set the [integrity metadata][] of the request to the value
-    of the element's `integrity` attribute.
-{:start="2"}
+Do a potentially CORS-enabled fetch of the resulting absolute URL, with the
+mode being the current state of the element's crossorigin content attribute,
```

Comment:
I'm not sure I parse this correctly.
Would this send credentials if the crossorigin attribute is "use-credentials"?

In any case, we still rely on CORS to cooperate, don't we?